songrde/src/main/java/com/de401/songrde/models/Album.java

```diff
@@ -1,19 +1,48 @@
-package com.de401.songrde;
+package com.de401.songrde.models;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
 public class Album {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    Long id;
     private String title;
     private String artist;
     private int songCount;
```

Comment:
Because this configuration has a protected default constructor, your entity's properties cannot be primitives. If the protected constructor were called, it would create a new Album with all properties set to `null`. However, `int`s are primitive and can't be null, so it's going to throw errors unless you use the wrapper class.